nx-X11/programs/Xserver/hw/nxagent/Display.c

```diff
@@ -2014,6 +2018,7 @@ Bool nxagentMakeIcon(Display *display, Pixmap *nxIcon, Pixmap *nxMask)
      }
   }
 
+  xfree(agentIconData);
```

Comment:
On  Di 22 Mär 2016 11:59:42 CET, Henning Heinold wrote:

>> @@ -2014,6 +2018,7 @@ Bool nxagentMakeIcon(Display *display, Pixmap  
>> *nxIcon, Pixmap *nxMask)
>>       }
>>    }
>>
>> +  xfree(agentIconData);
>
> Why did you use xfree here, instead of the normal free or is the  
> memcpy also from xserver?

Indeed. Using xfree is deprecated as of today. I will change this with  
a forced push.

Thanks for the first feedback.

Mike


-- 

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
fon: +49 (1520) 1976 148

GnuPG Key ID 0x25771B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de

freeBusy:
https://mail.das-netzwerkteam.de/mailxchange/kronolith/fb.php?u=m.gabriel%40das-netzwerkteam.de